content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+traffic to use a different version of the service for a specific user. To
+specify the user, the following example uses a regular expression to match the
+user from the cookie and shows you how to add the condition for the protocol in
+the `http:` field and a new `route:` and `destination:` configuration for the
+traffic matching the condition. In short, the example configures the HTTP
+traffic from the `jason` user to go to the `v2` subset of the `my-svc` service:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: my-vtl-svc
+spec:
+  hosts:
+    - *
+    - route:
+      - destination:
+        host: my-svc
+        subset: v1
+  http:
+    - match:
+      - headers:
+          cookie:
+            regex: "^(.*?;)?(user=jason)(;.*)?$"
+    route:
+      - destination:
+        host: my-svc
+        subset: v2
+{{< /text >}}
+
+The following diagram shows the configured traffic rules for both the traffic
+with the matched cookie and for all other traffic:
+
+{{< image width="50%"
+    link="./net-config-4.svg"
+    caption="Configurable traffic rules for traffic with and without a matched cookie"
+    >}}
+
+If you are working with a Kubernetes-based mesh, use virtual services to manage
+the traffic to your namespaces. The following example shows the `my-namespace`
+virtual service. It configures traffic routes for the two services behind the
+`my-namespace.svc.cluster.local` Kubernetes namespace:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: my-namespace
+spec:
+  hosts:
+    - *.my-namespace.svc.cluster.local
+  http:
+  - match:
+    - uri:
+        prefix: /svc-1
+    route:
+    - destination:
+        host: svc-1
+  - match:
+    - uri:
+        prefix: /svc-2
+    route:
+    - destination:
+        host: svc-2
+{{< /text >}}
+
+The following diagram shows the configured traffic routes implementing a
+virtual service to configure traffic for services in a namespace for two
+distinct application services based on the URI prefixes:
+
+{{< image width="50%"
+    link="./net-config-5.svg"
+    caption="Configurable traffic rules implementing a namespace for two  application services"
+    >}}
+
+Once you have identified and configured the virtual services suitable for your
+mesh, you can add gateways to route traffic in or out of your mesh. The virtual
+services together with destination rules allow you to configure the behavior of
+the traffic to fulfill the needs of your mesh. Together with a service entry,
+you can use virtual services to configure traffic routes to external
+dependencies.
+
+Visit our [virtual services reference documentation](/docs/reference/config/istio.networking.v1alpha3/#VirtualService) to review all the enabled keys and values.
+
+## Gateways
+
+The Istio gateways are layer 7 load balancers. With an Istio gateway, you can
+configure routes for HTTP and TCP traffic regardless of where the gateway runs.
```

Comment:
Your confusion is warranted, As @frankbu explained, gateway here is referring to the actual workload, which can run in a number of places. In a k8s mesh it would be in a pod. I'm changing the term "gateway" to "gateway workload" to avoid this confusion.